src/types/modules/react-native.d.ts

```diff
@@ -1,8 +1,343 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/no-empty-interface */
 /* eslint-disable @typescript-eslint/consistent-type-definitions */
+import {CSSProperties, FocusEventHandler, KeyboardEventHandler, MouseEventHandler, PointerEventHandler, UIEventHandler, WheelEventHandler} from 'react';
 import 'react-native';
 import {BootSplashModule} from '../../libs/BootSplash/types';
 
 declare module 'react-native' {
+    // <------ REACT NATIVE WEB (0.19.0) ------>
```

Comment:
I'm worried about the fact that we have to main these types.

Is there no other way but add/extend these types manually?

Reply:
React native web dont have typescript support so we need to write our own ts definition for web unfortunately.

Reply:
is there a community code or package that adds these types?

Reply:
@hayata-suenaga There is no package available that we know of. This should be handled internally in react native web library.

Reply:
okay thank you for letting me know